File: backend/router/dsl_history.go
package router

// DslHistory 路由
func (this *WebServer) runDslHistory() {

	const AbsolutePath = "/api/dslHistory"
	group := this.engine.Group("DSL历史记录", AbsolutePath)
	{
		group.POST(false, "查询自己的DSL历史记录", "/ListAction", this.esController.DslHistoryAction)
		group.POST(false, "清理自己的DSL历史记录", "/CleanAction", this.esController.CleanDslHistoryAction)
	}
}

File: frontend/src/api/dsl-history.ts
import {request} from "@elasticview/plugin-sdk";

const api = '/api/dslHistory/'

export function CleanAction(data) {
  return request({
    url: api + `CleanAction`,
    method: 'post',
    data
  })
}

export function ListAction(data) {
  return request({
    url: api + `ListAction`,
    method: 'post',
    data
  })
}


